bindings/nodejs/examples/how_tos/sign_ed25519/sign_ed25519.ts

```diff
@@ -0,0 +1,82 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import {
+    CoinType,
+    HD_WALLET_TYPE,
+    initLogger,
+    SecretManager,
+    utf8ToHex,
+    Utils,
+} from '@iota/sdk';
+require('dotenv').config({ path: '.env' });
+
+// In this example we will sign with Ed25519.
+// Run with command:
+// node ./dist/how_tos/sign_ed25519/sign_ed25519.js
```

Comment:
How does this work? Are there other steps before?

Reply:
yarn build/npm run build in the examples folder